pkg/csi/service/vanilla/controller.go

```diff
@@ -125,16 +127,19 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 		}
 		// Check if file service is enabled on datastore present in
 		// targetvSANFileShareDatastoreURLs.
-		dsToFileServiceEnabledMap, err := common.IsFileServiceEnabled(ctx, c.manager.VcenterConfig.TargetvSANFileShareDatastoreURLs, vc, datacenters)
+		dsToFileServiceEnabledMap, err := common.IsFileServiceEnabled(ctx,
+			c.manager.VcenterConfig.TargetvSANFileShareDatastoreURLs, vc, datacenters)
 		if err != nil {
-			msg := fmt.Sprintf("file service enablement check failed for datastore specified in TargetvSANFileShareDatastoreURLs. err=%v", err)
+			msg := fmt.Sprintf("file service enablement check failed for datastore specified in "+
```

Comment:
Shouldn't there be a space after " and before '+'?

Reply:
Golang seems preferring not having the white space. "Make check" will automatically remove it.

Reply:
Oh ok.